src/compas_blender/artists/boxartist.py

```diff
@@ -27,7 +29,7 @@ def __init__(self,
 
         super().__init__(shape=box, collection=collection or box.name, **kwargs)
 
-    def draw(self, color=None):
+    def draw(self, color: RGBColor = None) -> List[bpy.types.Object]:
```

Comment:
i think this would need to be `color: Optional[RGBColor] = None`

Reply:
Will change.